modules/sand/src/main/ts/ephox/sand/api/SandHTMLElement.ts

```diff
@@ -16,7 +18,9 @@ const isPrototypeOf = (x: any): x is HTMLElement => {
   // undefined scope later triggers using the global window.
   const scope: Window | undefined = Resolve.resolve('ownerDocument.defaultView', x);
```

Comment:
I'd rename `x` to something more meaningful. Element?!

Reply:
Not something changed in this PR.